src/main/resources/templates/menubar.html

```diff
@@ -1,13 +1,22 @@
 <div th:fragment="copy">
-<nav class="navbar fixed-top navbar-expand-lg navbar-light border-bottom pt-2 pb-2" th:fragment="copy">
+<nav class="navbar fixed-top navbar-expand-lg navbar-light border-bottom pt-2 pb-2" th:fragment="copy"
+     ng-controller="GeneralJsController">
     <div class="container">
         <a class="navbar-brand align-items-center" th:href="@{/}">
             <img class="img-fluid"
                  alt="UH Groupings Logo"
                  th:srcset="@{images/uh-groupings-logo.png 1x, images/uh-groupings-logo-2x.png 2x, images/uh-groupings-logo.svg}"
                  th:src="@{images/uh-groupings-logo.png}">
-            <span class="sr-only">UH Groupings</span>
-        </a>
+            <span class="sr-only">UH Groupings</span></a>
+        <span class="user-role rounded-circle seafoam-bg p-1 ml-5"
+              sec:authorize="hasAnyRole('ROLE_ADMIN', 'ROLE_OWNER')"
```

Comment:
Is there anyway to make this specify specifically for a departmental account? Because right now, the icon is showing for me although I am logged into my personal account. I think it would helpful to have a test departmental account that we can log in with.